todo-backend/app/Http/Controllers/TodoController.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Todo as Todo;
+
+class TodoController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index(Request $request)
+    {
+        $todos = Todo::all();
```

Comment:
nikako :scream: 
ovaj query ce ti povuci celu tabelu todos iz baze u ram
zamisli aplikaciju koju koristi milion korisnika i svaki ima 100 todo itema. to je 100 miliona todoova ucitanih u ram, aplikacija pada

mozes da uradis ovde `Todo::where('user_id', $userId)`, ali jos bolji pristup je da na user modelu i na todo modelu napravis relacije ka ovom drugom. onda bi mogao `$request->user()->todos()`

https://laravel.com/docs/5.8/eloquent-relationships